### DISCUSSION TITLE
Add cover position extra for tile

### DISCUSSION THREAD

**Discussion by LasseRosenow** [home-assistant/frontend#15184] [Dashboard] [answered]
Currently the light has a brightness slider on the tile card like this:

![203769467-caf27061-6a52-4993-b120-a80c260d8f02](https://user-images.githubusercontent.com/10547444/214333283-6a289e0d-2459-4199-af09-2b8befd3c6df.png)

It would be cool to be able to have this for the cover position also :)

**Comment by mgabor3141** [comment, marked as answer]
This has since been implemented:

<img width="493" alt="Screenshot 2023-09-09 at 9 27 17" src="https://github.com/home-assistant/frontend/assets/9047995/22694e85-206d-45b9-a924-7d779877621a">
